advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_library.py

```diff
@@ -16,7 +16,8 @@
 # with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-import roslib; #roslib.load_manifest('cyberglove')
+import roslib
+# roslib.load_manifest('cyberglove')
```

Comment:
How come this line is here and commented out? Surely should have explanation or be removed?